lib/index.js

```diff
@@ -252,6 +267,7 @@ export default {
 
   provideHyperclick(): HyperclickProvider {
     const provider = {
+      priority: atom.config.get('flow-ide.hyperclickPriority'),
```

Comment:
And this should use the property instead of having Atom run through the process of determining the config value again.